test/mesh_test.cpp

```diff
@@ -1053,7 +1053,8 @@ TEST(Boolean, Subtract) {
   first.GetMesh();
 }
 
-TEST(Boolean, Close) {
+// FIXME: test is failing on Mac CI (passing on others)
+TEST(Boolean, Close_DISABLED) {
```

Comment:
Sorry, meant `DISABLED_Close` - I always get that wrong...

Reply:
Ok that did the trick, test is being skipped so the mac build is "passing". :+1: